src/hooks.tsx

```diff
@@ -116,40 +116,49 @@ export const useAriaId = (id?: string): string => {
     }
 };
 
-export const useWindowSize = (): {
+const useSpecificWindowObject = (useInnerFrame: boolean): Window | undefined =>
```

Comment:
I'd avoid calling `useXXX` to functions that are not hooks. This function isn't a hook because it doesn't use any other hook (useState, useEffect, etc)

Reply:
🤔I see, I'd better leave it within the fixed footer layout itself, and call it `getSpecificWindowObject`. WDYT?